test/extended/dr/quorum_restore.go

```diff
@@ -188,7 +188,8 @@ var _ = g.Describe("[sig-etcd][Feature:DisasterRecovery][Disruptive]", func() {
 								return false, nil
 							}
 							if err != nil {
-								return false, err
+								framework.Logf("Error seen when re-creating machines: %v", err)
```

Comment:
This is a good start. Would it also make sense to apply this strategy of performing API operations in a poll loop and logging potentially recoverable errors rather than failing? i.e.

- ensure all calls to `scaleEtcdQuorum` occur in a poll loop
- ensure errors returned by `scaleEtcdQuorum` are logged rather than ignored
- ensure failing to list machines (L201) logs rather than fails on error
- ???

Ideally, all our e2e tests would be making the same assumptions as the controllers under test and be resilient to recoverable failure.

Reply:
It actually makes sense given the errors are recoverable and would reduce the number of false negatives. I will make the changes accordingly.